File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/model/domain/response/CommenterResponse.java
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.response;

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.Commenter;

public class CommenterResponse
{
  private String avatar_path;

  public Commenter getCommenter()
  {
    return new Commenter(avatar_path);
  }
}

File: app/src/main/java/io/github/kutaiba_nezar_kashmar/multiverse_of_geeks/model/domain/response/CommenterResponse.kt
package io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.response

import io.github.kutaiba_nezar_kashmar.multiverse_of_geeks.model.domain.local.Commenter

class CommenterResponse
{
    val avatar_path : String  = ""

    fun getCommenter() : Commenter
    {
        return Commenter(avatar_path)
    }
}